doc/content/getting-started/api/_index.md

```diff
@@ -0,0 +1,207 @@
+
+To register a device `newdev1` in application `app1`, first, register the `DevEUI`, `JoinEUI` and cluster addresses in the Identity Server. This is also where you register a friendly name, description, attributes, location, and more - see all fields in the [API Reference]({{< ref "reference/api" >}}):
+
+```bash
+$ curl --location \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' \
+  --header 'Content-Type: application/json' \
+  --header 'User-Agent: my-integration/my-integration-version' \
+  --request POST \
+  --data-raw '{
+    "end_device": {
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      },
+      "join_server_address": "thethings.example.com",
+      "network_server_address": "thethings.example.com",
+      "application_server_address": "thethings.example.com"
+    },
+    "field_mask": {
+      "paths": [
+        "join_server_address",
+        "network_server_address",
+        "application_server_address",
+        "ids.dev_eui",
+        "ids.join_eui"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/applications/app1/devices'
+```
+
+Register LoRaWAN settings in the Network Server:
+
+```bash
+$ curl --location \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+  --header 'User-Agent: my-integration/my-integration-version' \
+  --request PUT \
+  --data-raw '{
+    "end_device": {
+      "supports_join": true,
+      "lorawan_version": "1.0.2",
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      },
+      "lorawan_phy_version": "1.0.2-b",
+      "frequency_plan_id": "EU_863_870_TTN"
+    },
+    "field_mask": {
+      "paths": [
+        "supports_join",
+        "lorawan_version",
+        "ids.device_id",
+        "ids.dev_eui",
+        "ids.join_eui",
+        "lorawan_phy_version",
+        "frequency_plan_id"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/ns/applications/app1/devices/newdev1'
+```
+
+Register the `DevEUI` and `JoinEUI` in the Application Server:
+
+```bash
+$ curl --location \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+  --header 'User-Agent: my-integration/my-integration-version' \
+  --request PUT \
+  --data-raw '{
+    "end_device": {
+      "ids": {
+        "device_id": "newdev1",
+        "dev_eui": "XXXXXXXXXXXXXXXX",
+        "join_eui": "XXXXXXXXXXXXXXXX"
+      }
+    },
+    "field_mask": {
+      "paths": [
+        "ids.device_id",
+        "ids.dev_eui",
+        "ids.join_eui"
+      ]
+    }
+  }' \
+  'https://thethings.example.com/api/v3/as/applications/app1/devices/newdev1'
+```
+
+Finally, register the `DevEUI`, `JoinEUI`, cluster addresses, and keys in the Join Server:
```

Comment:
I guess it would be nice to mention that this is only relevant for OTAA devices?